en/ui-system/components/editor/webview.md

```diff
@@ -0,0 +1,210 @@
+
+```ts
+import { _decorator, Component, WebView } from 'cc';
+const { ccclass, property } = _decorator;
+
+@ccclass('cc.MyComponent')
+export class MyComponent extends Component {
+    @property(WebView)
+    webview = null;
+
+    start () {
+        const eventHandler = new Component.EventHandler();
+        eventHandler.target = newTarget; // This node is the one that the component that contains your event handler code belongs to
+        eventHandler.component = "cc.MyComponent";
+        eventHandler.handler = "callback";
+        eventHandler.customEventData = "foobar";
+        this.webview.webviewEvents.push(eventHandler);
+    }
+
+    // Note that the order and type of parameters are fixed
+    callback: function(webview, eventType, customEventData) {
+        // here webview is a WebView component instance
+        // here the value of eventType === cc.WebView.EventType enum
+        // The customEventData parameter here is equal to the "foobar"
+    }
+}
+```
+
+#### Method two
+
+Add event callback with `webview.node.on ('loaded', ...)`
+
+```ts
+// Suppose we add event handling callbacks in the onLoad method of a component and perform event handling in the callback function:
+import { _decorator, Component, WebView } from 'cc';
+const { ccclass, property } = _decorator;
+
+@ccclass('WebViewCtrl')
+export class WebViewCtrl extends Component {
+    @property(WebView)
+    webview = null;
+
+    start () {
+        this.webview.node.on('loaded', this.callback, this);
+    }
+
+    callback (event) {
+       // The event here is an EventCustom object, and you can get the WebView component through event.detail
+        let videoplayer = event.detail;
+        // do whatever you want with webview
+        // Also, note that this way the registered event can not pass customEventData
+    }
+}
+```
+
+Likewise, you can also register `loading`, `error` events, and the parameters of the callback function for these events are consistent with the `loaded` parameters.
+
+## How to interact with WebView internal pages
+
+### Calling the WebView internal page
+
+```ts
+import { _decorator, Component, WebView } from 'cc';
+const { ccclass, property } = _decorator;
+
+@ccclass('WebViewCtrl')
+export class WebViewCtrl extends Component {
+    @property(WebView)
+    webview = null;
+
+    start () {
+        // The Test here is a global function defined in your webView's internal page code
+        this.webview.evaluateJS('Test()');
+    }
+}
+```
+
+#### Note: Cross domain issues on Web platform need to be resolved by yourself
+
+### WebView internal pages call external code
+
+At present, the mechanism of Android and IOS is to determine whether the key of URL prefix is the same as that of the URL prefix by intercepting the jump, and then callback if the same.
```

Comment:
This paragraph doesn't make sense to me? 

mechanism? What does this mean?

intercepting the jump? What does this mean?